addons/autocontrol/maneuver.lua

```diff
@@ -91,10 +91,10 @@ Burden_tb = texts.new(str, settings)
 windower.register_event("action", function(act)
     if mjob_id == 18 then
         local abil_ID = act['param']
-        local actor_id = act['actor_id']
+        local actor_index = act['actor_index']
```

Comment:
Have not check `actor_index` exists but I assume it does.

Reply:
It does not.

Reply:
Shucks.